packages/camera/camera_android/android/src/test/java/io/flutter/plugins/camera/features/exposurepoint/ExposurePointFeatureTest.java

```diff
@@ -290,11 +292,11 @@ public void updateBuilder_shouldNotSetMeteringRectangleWhenNoValidBoundariesAreS
 
     exposurePointFeature.updateBuilder(mockCaptureRequestBuilder);
 
-    verify(mockCaptureRequestBuilder, times(1)).set(any(), isNull());
+    verify(mockCaptureRequestBuilder, times(1)).set(any(), any());
```

Comment:
Nit for this test and the next: if you make `defaultRegions` visible for testing, you should be able to test that the expected value is set versus `any()`

Reply:
done